### PR TITLE
pretty print

### PR DESCRIPTION
**Pull request by benjaminloerincz**

### CODE REVIEW

**Comment by patrickkettner** [comment]
thanks!

What did you use to pretty print? Ideally changes like this would include a new `npm run` command to automatically fix these issues project wide in the future

**Comment by benjaminloerincz** [comment]
I did not use npm to pretty print. I used prettier in vscode for that single manifest.json file manually.
I'm seeing that the npm script in this project does only pretty print *.md and *.html files, *.json files are ignored. If I run the script three additional files are changed:
* CONTRIBUTING.md
* README-template.md
* api-samples/printing/README.md

**Comment by patrickkettner** [comment]
I understand you did not use npm - it does not have the ability to do that. I am saying that adding a command to our package.json in order to make these sort of nit changes more sustainable long term.

**Comment by patrickkettner** [comment]
You can update the [`prettier` command in our scripts](https://github.com/GoogleChrome/chrome-extensions-samples/blob/ba108bff6ffeca4c515547c74db05bccff843773/package.json#L7) to include `json` files as well (i.e. `npx prettier **/*.{md,html,json} -w`).

After doing that, please rebase the PR on the current `main` branch and run the command and update this PR with those changes

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "private": true,
   "description": "Official samples for Chrome Extensions and the Chrome Apps platform.",
   "scripts": {
-    "prettier": "npx prettier **/*.{md,html} -w",
+    "prettier": "npx prettier **/*.{md,html,json} -w",
```
[the remark this answers is not in the document]
Comment (benjaminloerincz) [reply package.json:7]: done :)

**Review by patrickkettner — APPROVED** [review]
thanks!

**Pull request merged**